ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -686,6 +687,15 @@ export class AptosClient {
     return this.waitForTransactionWithResult(txnHash, extraArgs);
   }
 
+  @parseApiError
+  @Memoize({
+    ttlMs: 3600 * 1000, // cache result for 1h
```

Comment:
@gregnazario I thought this was cached on the API? Or should we also cache it here?

Reply:
1 hour is too long of a cache likely, @jjleng could you lower this down to say... 5 minutes?  It is cached on the API, but you could go to different nodes etc.  And this in theory could change very quickly

Reply:
I think we should cache it on the client side. This API was heavy. It took hundreds of milliseconds to finish.